specify_manual/command_design/show_annotattion/use_case.md

```
@@ -62,7 +66,7 @@ Document Anno [0]: @st-manual-spec-specification (Layer: Abstract, Type: Structu
 - **Steps:**
 
 ```bash
-$ strail show --mode list --target document
+$ strail show --mode list --target document --config src/config/simple_sample.toml
```

markdownlint-cli2 (0.22.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)